src/components/footer/footer.ts

```diff
@@ -13,33 +13,33 @@ export class FooterComponent extends LitElement {
     const { currentYear, STARTING_YEAR } = this;
 
     return html`
-     <footer class='container as-footer'>
+     <footer class="container as-footer">
         <section>
           <h2>Analog Studio's Footer</h2>
-          <article class='col-xs-12'>
+          <article class="col-xs-12">
            <h3>Copyright</h3>
-            <p class='copyright-text'>&copy; ${STARTING_YEAR} - ${currentYear} Analog Studios</p>
+            <p class="copyright-text">&copy; ${STARTING_YEAR} - ${currentYear} Analog Studios</p>
           </article>
-          <article class='col-xs-12'>
+          <article class="col-xs-12">
             <h3>Socials</h3>
-            <ul class='socials'>
+            <ul class="socials">
               <li>
-                <a href='https://www.facebook.com/analogstudiosri/'>
-                  <i class='fa fa-facebook-square' aria-hidden='true'>
+                <a href="https://www.facebook.com/analogstudiosri/">
+                  <i class="fa fa-facebook-square" aria-hidden="true">
```

Comment:
https://github.com/AnalogStudiosRI/www.analogstudios.net/pull/60#issuecomment-1005842041

Referencing applying <i> tags on HTML hardcoded in.